### ISSUE TITLE
improve SEO score

### ISSUE THREAD

**Issue by johannbarbie** [leapdao/leapdao-website#142]
<img width="1024" alt="Screen Shot 2020-01-08 at 5 09 50 PM" src="https://user-images.githubusercontent.com/659301/71995114-5859e600-323a-11ea-8630-67be1b8c5a6f.png">
from here: https://sitechecker.pro/

**Comment by saumyabratadutt** [comment]
Yes, the problem is meta description. It helps Google to rank site. But to exploit it further, if we add our niche related keywords to the description it can help further. Since we need to fix it anyway to rank.

**Comment by saumyabratadutt** [comment]
Some extra reports you might want to check from Gtmetrix, Woorank, Top pages from Moz.
https://drive.google.com/file/d/1zZ_3eezhcNAvzwbjsywVmOlCD-Ph9oiV/view?usp=sharing
https://drive.google.com/file/d/1KZ_cdfaqLCurA5IfBApeYHfZA29T7crw/view?usp=sharing
https://drive.google.com/file/d/1PXz55sCJ3wzbxlVkQl7NwsH56vWpBvAj/view?usp=sharing

**Comment by saumyabratadutt** [comment]
And the this is from Serpstat, shows some competitions etc
https://drive.google.com/file/d/1z-zh385WvLd0h0ZgzqwbhO_TsgmkGSZW/view?usp=sharing

**Comment by saumyabratadutt** [comment]
Issues from sitechecker.pro
https://i.ibb.co/GnZ60T0/Screenshot-1267.png

I think title tag, H1 & Title duplicates and description is the main problem, it needs to be fixed.
